src/Components/AddTask/AddTask.tsx

```diff
@@ -104,16 +151,21 @@ const AddTask: React.FC = () => {
           </div>
           <div className="row">
             <div className="col-md-6">
-                <select
+              <select
                 {...register("employeeId", { required: true, valueAsNumber: true })}
                 aria-label="Default select example"
-                type="number"
+                // type="number"
                 className="form-select"
               >
-                <option value="" className="text-muted">
+                <option  className="text-muted">
                   User
                 </option>
-                <option >name</option>
+                {userList.map((user)=>(
```

Comment:
you can improve this by destructuring the properties that you need from the user object directly
`userList.map(({id, userName})=>{...}`
and use it directly in the JSX that you want to return.